utils/_context/_scenarios.py

```diff
@@ -1100,14 +1100,32 @@ class scenarios:
     otel_tracing_e2e = OpenTelemetryScenario("OTEL_TRACING_E2E", doc="")
     otel_metric_e2e = OpenTelemetryScenario("OTEL_METRIC_E2E", include_intake=False, doc="")
     otel_log_e2e = OpenTelemetryScenario("OTEL_LOG_E2E", include_intake=False, include_agent=False, doc="")
-
+    # dummy value because the tests skip the first input.
     library_conf_custom_headers_short = EndToEndScenario(
-        "LIBRARY_CONF_CUSTOM_HEADERS_SHORT", additional_trace_header_tags=("header-tag1", "header-tag2"), doc=""
+        "LIBRARY_CONF_CUSTOM_HEADERS_SHORT", additional_trace_header_tags=("dummy", "header-tag1", "header-tag2"), doc="Scenario with <header_name> input given to DD_TRACE_HEADER_TAGS"
     )
     library_conf_custom_headers_long = EndToEndScenario(
         "LIBRARY_CONF_CUSTOM_HEADERS_LONG",
         additional_trace_header_tags=("header-tag1:custom.header-tag1", "header-tag2:custom.header-tag2"),
-        doc="",
+        doc="Scenario with <header>:<tag_name> formatted input given to DD_TRACE_HEADER_TAGS",
```

Comment:
Scenario are the most expensive stuff to add on system tests. Would it be doable to merge some of them ?